lib/core/logconf.c

```diff
@@ -262,6 +264,14 @@ h2o_logconf_t *h2o_logconf_compile(const char *fmt, int escape, char *errbuf)
                     MAP_EXT_TO_PROTO("ssl.cipher", ssl.cipher);
                     MAP_EXT_TO_PROTO("ssl.cipher-bits", ssl.cipher_bits);
                     MAP_EXT_TO_PROTO("ssl.session-id", ssl.session_id);
+
+                    h2o_log_handler_callback_t cb;
+                    if ((cb = h2o_proxy_get_logconf_callback(h2o_iovec_init(pt, quote_end - pt))) != NULL) {
+                        NEW_ELEMENT(ELEMENT_TYPE_HANDLER_SPECIFIC);
+                        LAST_ELEMENT()->data.handler_cb = cb;
+                        goto MAP_EXT_Found;
+                    }
+
```

Comment:
Can't we simply add `ELEMENT_TYPE_PROXY_PROCESS_TIME` etc. due to the same reason stated above?